docs/content/docs/operations/configuration.md

```diff
@@ -50,11 +50,39 @@ defaultConfiguration:
 
 To learn more about metrics and logging configuration please refer to the dedicated [docs page]({{< ref "docs/operations/metrics-logging" >}}).
 
+## Dynamic Operator Configuration
+
+The Kubernetes operator supports dynamic config changes through the operator ConfigMaps. Dynamic operator configuration is enabled by default, and can be disabled by setting `kubernetes.operator.dynamic.config.enabled`  to false. Time interval for checking dynamic config changes is specified by `kubernetes.operator.dynamic.config.check.interval` of which default value is 5 minutes. 
+
+Verify whether dynamic operator configuration updates is enabled via the `deploy/flink-kubernetes-operator` log has:
+
+```
+2022-05-28 13:08:29,222 o.a.f.k.o.c.FlinkConfigManager [INFO ] Enabled dynamic config updates, checking config changes every PT5M
+```
+
+When the interval for the controller to reschedule the reconcile process need to change with 30 seconds,  the ConfigMap can be directly edited via `kubectl patch` or `kubectl edit` command. The `kubernetes.operator.reconciler.reschedule.interval` config option, of which default value is 60 seconds, can directly change to 30s in the `defaultConfiguration` section:
+
+```yaml
+defaultConfiguration:
+  create: true
+  append: false
+  flink-conf.yaml: |+
+    # Flink Config Overrides
+    kubernetes.operator.reconciler.reschedule.interval: 30 s
+```
```

Comment:
Did you try this? I think updating the Helm values won't upgrade the configmap (if the operator is already running)

So please  change this to simply:

```
To change config values dynamically the ConfigMap can be directly edited via `kubectl patch` or `kubectl edit` command. For example to change the reschedule interval you can override `kubernetes.operator.reconciler.reschedule.interval`.
```